shibboleth-sp-singlevm-centos/azuredeploy.json

```diff
@@ -0,0 +1,214 @@
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "String",
+            "metadata": {
+                "description": "User name for the Virtual Machine administrator. Do not use simple names such as 'admin'."
+            }
+        },
+        "adminPassword": {
+            "type": "SecureString",
+            "metadata": {
+                "description": "Password for the Virtual Machine administrator."
+            }
+        },
+        "uniqueNamePrefix": {
+            "type": "String",
+            "metadata": {
+                "description": "Unique name that will be used to generate various other names including the name of the Public IP used to access the Virtual Machine."
+            }
+        },
+        "vmSize": {
+            "defaultValue": "Standard_A0",
+            "allowedValues": [
+                "Standard_A0",
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_A5",
+                "Standard_A6",
+                "Standard_A7",
+                "Standard_A8",
+                "Standard_A9",
+                "Standard_A10",
+                "Standard_A11",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4",
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14"
+            ],
+            "type": "String",
+            "metadata": {
+                "description": "The size of the VM."
+            }
+        }
+    },
+    "variables": {
+        "newStorageAccountName": "[concat(uniquestring(resourceGroup().id), 'shs')]",
+        "imagePublisher": "OpenLogic",
+        "imageOffer": "CentOS",
+        "imageSKU": "7.0",
+        "location": "[resourceGroup().location]",
+        "OSDiskName": "[concat(parameters('uniqueNamePrefix'),'Disk')]",
+        "nicName": "[concat(parameters('uniqueNamePrefix'),'Nic')]",
+        "addressPrefix": "10.0.0.0/16",
+        "subnetName": "Subnet",
+        "subnetPrefix": "10.0.0.0/24",
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressName": "[concat(parameters('uniqueNamePrefix'),'IP')]",
+        "publicIPAddressType": "Dynamic",
+        "vmStorageAccountContainerName": "vhds",
+        "vmName": "[concat(parameters('uniqueNamePrefix'),'VM')]",
+        "virtualNetworkName": "[concat(parameters('uniqueNamePrefix'),'VNet')]",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+        "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+        "installScriptName": "install_shibboleth_sp.sh",
+        "rawUrl" : "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/shibboleth-sp-singlevm-centos/",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts